share/scan/apps_test.go

```diff
@@ -275,3 +275,37 @@ Bundle-ActivationPolicy: lazy
 		t.Errorf("Wrong jar package: %+v\n", pkg)
 	}
 }
+
+func TestGetDotnetModuleName(t *testing.T) {
+	incomingName := "Microsoft.AspNetCore.App.Runtime.linux-musl-x64/6.0.15"
```

Comment:
how about change this into a slices, like following struct, which improve readability and easy to add new tasks
```go
tests := []struct {
    outgoingName string
    expectedName string
}{
    {"module1", "module1"},
    {"module2", "expectedModule2"},
}
```